File: zhilly11/BOJ/Greedy/11399_ATM.swift
//
//  [백준] 11399.swift
//  CodingStudy
//
//  Created by 최지혁 on 2022/05/10.
//

import Foundation

let N = Int(readLine()!)!
var time : [Int] = Array(repeating: 0, count: N)

time = readLine()!.split(separator: " ").map { Int(String($0))! }

time.sort()

var result = 0

for i in 0...N {
    for j in 0..<i {
        result += time[j]
    }
}

print(result)

File: spade8/2nd_week/11399_atm.py
import sys

people_num = int(sys.stdin.readline())
people = list(map(int, sys.stdin.readline().split()))

result = 0
##데이터 입력
people.sort(reverse=True)

for i in range(people_num):
    result = result + (i + 1) * people[i]

print(result)